notebooks/MAST/Identifying-transient-candidates/identifying_highz_candidates.ipynb

```diff
@@ -0,0 +1,598 @@
+{
```

Comment:
<p>It would be good to include a table of contents / workflow following this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>